_episodes/03-changes.md

```diff
@@ -298,23 +295,22 @@ $ git commit -m "readme: where are the files?"
 >     From 7248345bacf4593e35c24398a4c375815ee006fc Mon Sep 17 00:00:00 2001
 >     From: Catsy Cline <ccline@pawson.edu>
 >     Date: Mon, 2 Oct 2017 12:43:54 -0700
->     Subject: [PATCH] readme: where are the files?
->
->     ---
->      README.md | 2 ++
->      1 file changed, 2 insertions(+)
+>Subject: [PATCH] readme: where are the files?
```

Comment:
This needs to be indented at the same level as the rest of the patch.

Reply:
fixed, let me know what you think.